entity-management-mongo/src/main/java/eu/europeana/entitymanagement/mongo/repository/EntityRecordRepository.java

```diff
@@ -29,7 +30,7 @@
      * @return the total number of resources in the database
      */
     public long count() {
-        return datastore.find(EntityRecordImpl.class).count();
+        return datastore.find(BaseEntityRecord.class).count();
```

Comment:
The type of the objects in the database must be the EntityRecordImpl not the BaseEntityRecord

Reply:
We have discussed to keep this so far as it is.